.travis.yml

```diff
@@ -6,13 +6,10 @@ jobs:
   include:
     - addons:
         chrome: stable
-      env: BROWSER=ChromeCi
+      env: BROWSER=ChromeHeadless
```

Comment:
The GH Actions CI is already running a Chrome instance, yeah? IMO we should just remove the Travis config altogether :stuck_out_tongue:

Reply:
maybe let it there until GitHub Actions is out of Beta?

Reply:
maybe drop firefox from travis, then? and just keep chrome there?